authentication/views.py

```diff
@@ -77,3 +93,40 @@ def post(self, request):
             print("Exception", e)
             return Response(status=status.HTTP_400_BAD_REQUEST)
 
+
+class PasswordRecoveryAPIView(APIView):
+    def post(self, request):
+        email = request.data.get('email')
+        try:
+            validate_email(email)
+        except ValidationError:
+            return Response({'error': 'Invalid email address'}, status=400)
+
+        try:
+            user = CustomUser.objects.get(email=email)
```

Comment:
*Just a piece of advice:*
Consider the user experience and security implications of revealing whether an email is registered in the system. Sometimes, it's better to always return a successful response to prevent email enumeration attacks, where an attacker could use this endpoint to find out which emails are registered on your platform

Reply:
Thank you, changing error description to standart "Bad request". This conceals information about the presence or absence of emails from potential attackers.